contracts/validator-manager/interfaces/IValidatorManager.sol

```diff
@@ -77,6 +77,10 @@ interface IValidatorManager is IACP99Manager {
         uint64 newWeight
     ) external returns (uint64, bytes32);
 
+    function transferOwnership(
```

Comment:
I don't think we should need to add `transferOwnership` to the interface here, since it's valid for implementations to not have owners theoretically. Is there a reason why you added it here? 

The implementation of `ValidatorManager` already has `transferOwnership` from the parent `OwnerUpgradeable` contract [here](https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/access/OwnableUpgradeable.sol#L102).